components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp

```diff
@@ -3744,7 +3747,7 @@ function doValidation() {
             <td>
                 <div class="sectionCheckbox">
                     <input id="IsEnableAssetionEncription" name="IsEnableAssetionEncription"
-                           type="checkbox" <%=enableAssertinEncriptionChecked%>/>
+                           type="checkbox" <%=Encode.forHtmlAttribute(enableAssertinEncriptionChecked)%>/>
```

Comment:
The variable value is hardcoded. Further this is not an attribute. The variable fully represents the attribute key and value here. So no need to encode.